base_exception/models/base_exception.py

```diff
@@ -202,6 +218,14 @@ def _compute_main_error(self):
             else:
                 rec.main_exception_id = False
 
+    @api.depends('exception_ids', 'ignore_exception')
+    def _compute_exception_ids_formatted(self):
+        for rec in self:
+            if rec.exception_ids and not rec.ignore_exception:
+                rec.exception_ids_formatted = '\r\n'.join([
```

Comment:
Would it make sense to make this field of type `Html` and join it in a `<ul><li>...</li>` way? IMHO it would look better.

Reply:
Yes, I thought about that but in fact I couldn't find a way to render the raw html on the message. (https://github.com/OCA/sale-workflow/pull/878/files#diff-e3019ca8759cdf323e38ee35b80b147cR33)
If you know an easy and clean way let me know

Reply:
Yes, just use an html field as I said :sweat_smile:

Reply:
Are you going to apply this, or do we merge?

Reply:
I'll try it on Monday ;) I made this PR a while ago but I remember trying to use html at the time, I just don't remember what the issue was. Anyways it's worth trying again

Reply:
Well.. I honestly don't remember what I tried 😅 but it's working